Server/crafting.txt

```diff
@@ -110,55 +113,58 @@ TNT                     = Gunpowder, 1:1, 3:1, 2:2, 1:3, 3:3 | Sand, 2:1, 1:2, 3
-BirchWoodSlab,   6  = BirchPlanks,       1:1, 2:1, 3:1
-BrickSlab,       6  = BrickBlock,        1:1, 2:1, 3:1
-CobblestoneSlab, 6  = Cobblestone,       1:1, 2:1, 3:1
-DarkOakWoodSlab, 6  = DarkOakPlanks,     1:1, 2:1, 3:1
-JungleWoodSlab,  6  = JunglePlanks,      1:1, 2:1, 3:1
-NetherbrickSlab, 6  = NetherBrick,       1:1, 2:1, 3:1
-OakWoodSlab,     6  = OakPlanks,         1:1, 2:1, 3:1
-Quartzslab,      6  = QuartzBlock,       1:1, 2:1, 3:1
+AcaciaWoodSlab,   6 = AcaciaPlanks,      1:1, 2:1, 3:1
+BirchWoodSlab,    6 = BirchPlanks,       1:1, 2:1, 3:1
+BrickSlab,        6 = BrickBlock,        1:1, 2:1, 3:1
+CobblestoneSlab,  6 = Cobblestone,       1:1, 2:1, 3:1
+DarkOakWoodSlab,  6 = DarkOakPlanks,     1:1, 2:1, 3:1
+JungleWoodSlab,   6 = JunglePlanks,      1:1, 2:1, 3:1
+NetherbrickSlab,  6 = NetherBrick,       1:1, 2:1, 3:1
+OakWoodSlab,      6 = OakPlanks,         1:1, 2:1, 3:1
+PurpurSlab,       6 = PurpurBlock,       1:1, 2:1, 3:1
+Quartzslab,       6 = QuartzBlock,       1:1, 2:1, 3:1
 RedSandstoneSlab, 6 = RedSandstone^-1,   1:1, 2:1, 3:1
-SandstoneSlab,   6  = OrnamentSandstone, 1:1, 2:1, 3:1
-SandstoneSlab,   6  = Sandstone,         1:1, 2:1, 3:1
-SandstoneSlab,   6  = SmoothSandstone,   1:1, 2:1, 3:1
-SnowLayer,       6  = SnowBlock,         1:1, 2:1, 3:1
-SpruceWoodSlab,  6  = SprucePlanks,      1:1, 2:1, 3:1
-StonebrickSlab,  6  = StoneBrick,        1:1, 2:1, 3:1
-StoneSlab,       6  = Stone,             1:1, 2:1, 3:1
+SandstoneSlab,    6 = OrnamentSandstone, 1:1, 2:1, 3:1
+SandstoneSlab,    6 = Sandstone,         1:1, 2:1, 3:1
+SandstoneSlab,    6 = SmoothSandstone,   1:1, 2:1, 3:1
+SnowLayer,        6 = SnowBlock,         1:1, 2:1, 3:1
+SpruceWoodSlab,   6 = SprucePlanks,      1:1, 2:1, 3:1
+StonebrickSlab,   6 = StoneBrick,        1:1, 2:1, 3:1
+StoneSlab,        6 = Stone,             1:1, 2:1, 3:1
 
 # Stairs:
-AcaciaWoodStairs, 4  = AcaciaPlanks, 1:1, 1:2, 2:2, 1:3, 2:3, 3:3
-AcaciaWoodStairs, 4  = AcaciaPlanks, 3:1, 2:2, 3:2, 1:3, 2:3, 3:3
-BirchWoodStairs, 4   = BirchPlanks, 1:1, 1:2, 2:2, 1:3, 2:3, 3:3
-BirchWoodStairs, 4   = BirchPlanks, 3:1, 2:2, 3:2, 1:3, 2:3, 3:3
-BrickStairs, 4       = BrickBlock,  1:1, 1:2, 2:2, 1:3, 2:3, 3:3
-BrickStairs, 4       = BrickBlock,  3:1, 2:2, 3:2, 1:3, 2:3, 3:3
-cobblestoneStairs, 4 = Cobblestone, 1:1, 1:2, 2:2, 1:3, 2:3, 3:3
-cobblestoneStairs, 4 = Cobblestone, 3:1, 2:2, 3:2, 1:3, 2:3, 3:3
-DarkOakWoodStairs, 4 = DarkOakPlanks, 1:1, 1:2, 2:2, 1:3, 2:3, 3:3
-DarkOakWoodStairs, 4 = DarkOakPlanks, 3:1, 2:2, 3:2, 1:3, 2:3, 3:3
-JungleWoodStairs, 4  = JunglePlanks, 1:1, 1:2, 2:2, 1:3, 2:3, 3:3
-JungleWoodStairs, 4  = JunglePlanks, 3:1, 2:2, 3:2, 1:3, 2:3, 3:3
-NetherBrickStairs, 4 = NetherBrick, 1:1, 1:2, 2:2, 1:3, 2:3, 3:3
-NetherBrickStairs, 4 = NetherBrick, 3:1, 2:2, 3:2, 1:3, 2:3, 3:3
-quartzstairs, 4      = QuartzBlock, 1:1, 1:2, 2:2, 1:3, 2:3, 3:3
-quartzstairs, 4      = QuartzBlock, 3:1, 2:2, 3:2, 1:3, 2:3, 3:3
+AcaciaWoodStairs, 4   = AcaciaPlanks, 1:1, 1:2, 2:2, 1:3, 2:3, 3:3
```

Comment:
Only the = signs are aligned below this line. I mean that the numbers should also be aligned